### PR TITLE
Add management of voltage init's default voltage limits parameters

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/components/dialogs/parameters/voltageinit/voltage-init-parameters.js [hunk 1]
```diff
@@ -95,6 +96,41 @@ const formSchema = yup.object().shape({
...
+                .min(1, 'FilterInputMinError'),
+            [LOW_VOLTAGE_LIMIT]: yup
+                .number()
+                .min(0)
+                .nullable()
+                .test((value, context) => {
+                    return (
+                        (value !== null && value !== undefined) ||
+                        (context.parent[HIGH_VOLTAGE_LIMIT] !== null &&
+                            context.parent[HIGH_VOLTAGE_LIMIT] !== undefined)
```
Comment (ayolab) [inline src/components/dialogs/parameters/voltageinit/voltage-init-parameters.js:118]:
```suggestion
                        !isBlankOrEmpty(value) ||
                        !isBlankOrEmpty(context.parent[HIGH_VOLTAGE_LIMIT])
```

File: src/components/dialogs/parameters/voltageinit/voltage-init-parameters.js [hunk 2]
```diff
@@ -95,6 +96,41 @@ const formSchema = yup.object().shape({
...
+                }),
+            [HIGH_VOLTAGE_LIMIT]: yup
+                .number()
+                .min(0)
+                .nullable()
+                .test((value, context) => {
+                    return (
+                        (value !== null && value !== undefined) ||
+                        (context.parent[LOW_VOLTAGE_LIMIT] !== null &&
+                            context.parent[LOW_VOLTAGE_LIMIT] !== undefined)
```
Comment (ayolab) [inline src/components/dialogs/parameters/voltageinit/voltage-init-parameters.js:129]:
```suggestion
                        !isBlankOrEmpty(value) ||
                        !isBlankOrEmpty(context.parent[LOW_VOLTAGE_LIMIT])
```

File: src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js [hunk 3]
```diff
@@ -78,28 +74,107 @@ const VoltageLimitsParameters = () => {
...
-            arrayFormName={`${VOLTAGE_LIMITS}`}
-            columnsDefinition={COLUMNS_DEFINITIONS}
-            useFieldArrayOutput={useFieldArrayOutput}
-            createRows={createVoltageLimitRows}
-            tableHeight={270}
-            withAddRowsDialog={false}
-            withLeftButtons={false}
-        />
+        <>
+            <Typography component="span" variant="h6">
```
Comment (sBouzols) [inline src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js:148]: Why not having a DialogContent here like in `EquipmentSelectionParameters` and a Grid Layout with some titles like other places in parameters ? I mean it's the only reference to a Typography h6 in the code.

File: src/translations/en.json [hunk 4]
```diff
@@ -159,6 +159,8 @@
     "startVoltageInitError": "An error occurred while starting the voltage profile initialization",
     "VoltageLimits": "Voltage limits",
     "VoltageLevelFilter": "Voltage level filter",
+    "LowVoltageLimitDefault": "Low voltage limit default value",
+    "HighVoltageLimitDefault": "High voltage default value",
```
Comment (sBouzols) [inline src/translations/en.json:163]:
```suggestion
    "HighVoltageLimitDefault": "High voltage limit default value",
```

File: src/translations/en.json [hunk 5]
```diff
@@ -169,6 +171,8 @@
     "VariableShuntCompensators": "Variable shunt compensators",
     "voltageInitError": "An error occurred while excuting the voltage profile initialization",
     "VoltageInitParametersError": "An error occurred while updating the voltage profile initialization parameters",
+    "AdjustExistingLimits": "Adjust existing limits",
+    "SetDefaultLimits" : "Defaults values to fill missing limits",
```
Comment (sBouzols) [inline src/translations/en.json:175]:
```suggestion
    "SetDefaultLimits" : "Default values to fill missing limits",
```

Comment (Meklo) [reply src/translations/en.json]: Done

File: src/translations/en.json (hunk 4 above)
Comment (Meklo) [reply src/translations/en.json]: Done

File: src/components/dialogs/parameters/voltageinit/voltage-init-parameters.js (hunk 2 above)
Comment (Meklo) [reply src/components/dialogs/parameters/voltageinit/voltage-init-parameters.js]: Done

File: src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js (hunk 3 above)
Comment (Meklo) [reply src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js:148]: I'll try it out

Comment (Meklo) [reply src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js]: I've replaced Typography with some Grid component combined with css

**Review by ayolab — APPROVED** [review]

**Review by sBouzols — APPROVED** [review]
Code Review OK
Tests done OK
Console warning check OK

**Pull request merged**